 src/target/cortex_m.c |    5 +++++
 1 file changed, 5 insertions(+)

diff --git a/src/target/cortex_m.c b/src/target/cortex_m.c
index 5fa78ca..8f232b5 100644
--- a/src/target/cortex_m.c
+++ b/src/target/cortex_m.c
@@ -1570,6 +1570,11 @@ static int cortex_m3_read_memory(struct target *target, uint32_t address,
 	struct adiv5_dap *swjdp = armv7m->arm.dap;
 	int retval = ERROR_COMMAND_SYNTAX_ERROR;
 
+	if ((address & 0xFFFFFFE0) == 0xFFFFFFE0)
+	{
+		return retval;
+	}
+	
 	if (armv7m->arm.is_armv6m) {
 		/* armv6m does not handle unaligned memory access */
 		if (((size == 4) && (address & 0x3u)) || ((size == 2) && (address & 0x1u)))
